webpack/scenes/ContentViews/Details/ContentViewDetailActions.js

```diff
@@ -63,4 +64,11 @@ export const getContentViewRepositories = (cvId, params, status) => {
   });
 };
 
+export const getRepositoryTypes = () => async dispatch => dispatch(get({
```

Comment:
In general `dispatch` shouldn't be used in action creators.  I've never liked that feature of `redux-thunk` because it makes it too easy to muddle the purposes of the different parts of Redux.  I don't think this is needed anyway -- you can just return the action itself and dispatch it in the component.  This way your action creator doesn't need to be a thunk, and can become a simple action instead:

```suggestion
export const getRepositoryTypes => get({
```
And the API middleware will take care of the rest :)

Reply:
I forgot we have this now https://github.com/theforeman/foreman/pull/7841 - I can update!